torch/csrc/jit/interpreter.cpp

```diff
@@ -193,6 +362,55 @@ struct CodeImpl {
   std::vector<bool> bool_data;
 };
 
+
+struct Registers {
+  Registers(size_t size)
+  : registers(size) {}
+  Registers(const Registers & rhs)
+  : registers(rhs.registers) {
+    for(auto & r : registers) {
+      if(isValid(r))
+        r->retain();
```

Comment:
Doesn't `refcounted_list` automatically bump the refcount when you copy it?

Reply:
I renamed it list_of_refcounted because refcounted_list is highly ambiguous.